x-pack/plugins/cross_cluster_replication/server/lib/register_license_checker/register_license_checker.js

```diff
@@ -5,10 +5,10 @@
  */
```

Comment:
looks like the header in this file is duplicated, can we fix this while we're at it? 😄

Reply:
Good catch! Done.